tests/unit/accordion/accordion_methods.js

```diff
@@ -7,10 +7,17 @@ var equalHeight = TestHelpers.accordion.equalHeight,
 module( "accordion: methods", setupTeardown() );
 
 test( "destroy", function() {
-	expect( 1 );
+	expect( 2 );
 	domEqual( "#list1", function() {
 		$( "#list1" ).accordion().accordion( "destroy" );
 	});
+
+	// Animate
+	domEqual( "#list1", function() {
+		$( "#list1" ).accordion()
+			.accordion( "option", "active", 1 )
+			.accordion( "destroy" );
+	});
```

Comment:
Is this actually related to the classes implementation?

This isn't doing what you expect anyway, since animations are disabled in tests (see line 7).